apps/vanilla/src/pages/blackJack/models/turnGenerator.ts

```diff
@@ -0,0 +1,24 @@
+import { Publisher } from './publisher';
+const START_INDEX = -1;
+
+/** Turn player index with publsher func. */
+export class TurnGenerator extends Publisher<number> {
+
+	public constructor(private readonly playerCount = 2) {
+		super();
+		this.playerCount = playerCount;
+	}
```

Comment:
Property assignment inside the constructor is redundant. See [Parameter Properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties):
```suggestion
	public constructor(private readonly playerCount = 2) {
		super();
	}
```